src/Evolution/Systems/GrMhd/ValenciaDivClean/FiniteDifference/ReconstructWork.tpp

```diff
@@ -193,17 +187,11 @@ void reconstruct_prims_work(
              "got "
                  << neighbors_in_direction.size() << " in direction "
                  << direction);
-      ASSERT(not neighbor_data
-                     .at(std::pair{direction, *neighbors_in_direction.begin()})
-                     .empty(),
-             "The neighber data is empty in direction "
-                 << direction << " on element id " << element.id());
-      ghost_cell_vars[direction] = gsl::make_span(
-          &neighbor_data.at(std::pair{
-              direction,
-              *neighbors_in_direction
-                   .begin()})[vars_in_neighbor_count * neighbor_num_pts],
-          number_of_variables * neighbor_num_pts);
+      ghost_cell_vars[direction] =
```

Comment:
Because this now gets called twice: once for tags that need to be kept positive and once for the remaining tags. This means the `ASSERT` will always fail the second time